Makefile

```diff
@@ -0,0 +1,50 @@
+IMAGE_NAME=blueoil_$$(id -un)
+BUILD_VERSION=$(shell git describe --tags --always --dirty --match="v*" 2> /dev/null || cat $(CURDIR/.version 2> /dev/null || echo v0))
+
+default: build
+
+.PHONY: deps
+deps:
+	# update dependencies
+	git submodule update --init --recursive
+
+.PHONY: build
+build: deps
+	# build docker container
+	docker build -t $(IMAGE_NAME):$(BUILD_VERSION) --build-arg python_version="3.6.3" -f docker/Dockerfile .
+
+.PHONY: test
+test: build
+	# run test
+	bash ./blueoil_test.sh
+
+.PHONY: build-lmnet
+build-lmnet:
+	# build lmnet docker containner
+	cd lmnet; docker-compose build --pull --no-cache tensorflow > /dev/null
+
+.PHONY: build-coco
+build-coco: build-lmnet
+	# build coco PythonAPI
+	cd lmnet; docker-compose run --rm tensorflow /bin/bash -cx "cd third_party/coco/PythonAPI && make -j8"
+
+.PHONY: test-lmnet
+test-lmnet: build-coco
+	# run lmnet test with Python3.5
+	cd lmnet; docker-compose run --rm tensorflow tox -e py35
+
+.PHONY: test-dlk
+test-dlk: build
+	# run dlk test
+	docker run --rm -t -v /root/.ssh:/root/.ssh --net=host $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "apt-get update && apt-get install -y iputils-ping && cd dlk && python setup.py test"
+
+.PHONY: pep8-dlk
+pep8-dlk:
+	# check dlk PEP8
+	docker run --rm -t $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cd dlk && pycodestyle --ignore=W --max-line-length=120 --exclude='*static/pb*','*docs/*','*.eggs*','*tvm/*','*tests/*','backends/*' ."
+
+.PHONY: clean
+clean:
```

Comment:
I think `make clean` also shoud remove `lmnet_tensorflow`.

Reply:
now lmnet_tensorflow image is not created. so I skip this.